keyboards/zeix/singa/kohaku/keyboard.json

```diff
@@ -0,0 +1,717 @@
+                {"matrix": [2, 1], "x": 2.5, "y": 1},
+                {"matrix": [3, 1], "x": 3.5, "y": 1},
+                {"matrix": [2, 2], "x": 4.5, "y": 1},
+                {"matrix": [3, 2], "x": 5.5, "y": 1},
+                {"matrix": [2, 3], "x": 6.5, "y": 1},
+                {"matrix": [3, 3], "x": 7.5, "y": 1},
+                {"matrix": [2, 4], "x": 8.5, "y": 1},
+                {"matrix": [3, 4], "x": 9.5, "y": 1},
+                {"matrix": [2, 5], "x": 10.5, "y": 1},
+                {"matrix": [3, 5], "x": 11.5, "y": 1},
+                {"matrix": [2, 6], "x": 12.5, "y": 1},
+                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
+                {"matrix": [3, 7], "x": 15, "y": 1},
+                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
+                {"matrix": [5, 0], "x": 1.75, "y": 2},
+                {"matrix": [4, 1], "x": 2.75, "y": 2},
+                {"matrix": [5, 1], "x": 3.75, "y": 2},
+                {"matrix": [4, 2], "x": 4.75, "y": 2},
+                {"matrix": [5, 2], "x": 5.75, "y": 2},
+                {"matrix": [4, 3], "x": 6.75, "y": 2},
+                {"matrix": [5, 3], "x": 7.75, "y": 2},
+                {"matrix": [4, 4], "x": 8.75, "y": 2},
+                {"matrix": [5, 4], "x": 9.75, "y": 2},
+                {"matrix": [4, 5], "x": 10.75, "y": 2},
+                {"matrix": [5, 5], "x": 11.75, "y": 2},
+                {"matrix": [4, 6], "x": 12.75, "y": 2},
+                {"matrix": [5, 6], "x": 13.75, "y": 2, "w": 1.25, "h": 2},
+                {"matrix": [5, 7], "x": 15, "y": 2},
+                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
+                {"matrix": [7, 0], "x": 1.25, "y": 3},
+                {"matrix": [6, 1], "x": 2.25, "y": 3},
+                {"matrix": [7, 1], "x": 3.25, "y": 3},
+                {"matrix": [6, 2], "x": 4.25, "y": 3},
+                {"matrix": [7, 2], "x": 5.25, "y": 3},
+                {"matrix": [6, 3], "x": 6.25, "y": 3},
+                {"matrix": [7, 3], "x": 7.25, "y": 3},
+                {"matrix": [6, 4], "x": 8.25, "y": 3},
+                {"matrix": [7, 4], "x": 9.25, "y": 3},
+                {"matrix": [6, 5], "x": 10.25, "y": 3},
+                {"matrix": [7, 5], "x": 11.25, "y": 3},
+                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
+                {"matrix": [7, 6], "x": 14, "y": 3},
+                {"matrix": [7, 7], "x": 15, "y": 3},
+                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.5},
+                {"matrix": [9, 0], "x": 1.5, "y": 4, "w": 1},
+                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.5},
+                {"matrix": [8, 3], "x": 4, "y": 4, "w": 7},
+                {"matrix": [8, 5], "x": 11, "y": 4, "w": 1.5},
+                {"matrix": [8, 6], "x": 13, "y": 4},
+                {"matrix": [9, 6], "x": 14, "y": 4},
+                {"matrix": [9, 7], "x": 15, "y": 4}
+            ]
+        }
+    }
+}
```

Comment:
- fixed erroneous key data
  - changed `h` to `w`
  - Enter key width, for `LAYOUT_all` layout, shortened
  - for ANSI layouts, corrected left shift width
  - for ISO layouts, changed ISO Enter row number (`y`) to match Community Layout
  - fixed ANSI tsangan layout, was using 6.25/blocker key information 
- rename layout name(s) to match respective Community Layout(s)
```suggestion
        "LAYOUT_all": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [1, 6], "x": 13, "y": 0},
                {"matrix": [0, 7], "x": 14, "y": 0},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [4, 6], "x": 12.75, "y": 2},
                {"matrix": [5, 6], "x": 13.75, "y": 2, "w": 1.25},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [7, 0], "x": 1.25, "y": 3},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [9, 2], "x": 3.75, "y": 4, "w": 2.75},
                {"matrix": [8, 3], "x": 6.5, "y": 4, "w": 1.25},
                {"matrix": [9, 4], "x": 7.75, "y": 4, "w": 2.25},
                {"matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [0, 7], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [5, 6], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [8, 3], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [1, 6], "x": 13, "y": 0},
                {"matrix": [0, 7], "x": 14, "y": 0},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [5, 6], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [8, 3], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker_tsangan": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [0, 7], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [5, 6], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [9, 0], "x": 1.5, "y": 4, "w": 1},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [8, 3], "x": 4, "y": 4, "w": 7},
                {"matrix": [8, 5], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_ansi_blocker_tsangan_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [1, 6], "x": 13, "y": 0},
                {"matrix": [0, 7], "x": 14, "y": 0},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 6], "x": 13.5, "y": 1, "w": 1.5},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [5, 6], "x": 12.75, "y": 2, "w": 2.25},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 2.25},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [9, 0], "x": 1.5, "y": 4, "w": 1},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [8, 3], "x": 4, "y": 4, "w": 7},
                {"matrix": [8, 5], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [0, 7], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [4, 6], "x": 12.75, "y": 2},
                {"matrix": [5, 6], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [7, 0], "x": 1.25, "y": 3},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [8, 3], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [1, 6], "x": 13, "y": 0},
                {"matrix": [0, 7], "x": 14, "y": 0},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [4, 6], "x": 12.75, "y": 2},
                {"matrix": [5, 6], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [7, 0], "x": 1.25, "y": 3},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.25},
                {"matrix": [9, 0], "x": 1.25, "y": 4, "w": 1.25},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.25},
                {"matrix": [8, 3], "x": 3.75, "y": 4, "w": 6.25},
                {"matrix": [8, 5], "x": 10, "y": 4, "w": 1.25},
                {"matrix": [9, 5], "x": 11.25, "y": 4, "w": 1.25},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker_tsangan": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [0, 7], "x": 13, "y": 0, "w": 2},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [4, 6], "x": 12.75, "y": 2},
                {"matrix": [5, 6], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [7, 0], "x": 1.25, "y": 3},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [9, 0], "x": 1.5, "y": 4, "w": 1},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [8, 3], "x": 4, "y": 4, "w": 7},
                {"matrix": [8, 5], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        },
        "LAYOUT_65_iso_blocker_tsangan_split_bs": {
            "layout": [
                {"matrix": [0, 0], "x": 0, "y": 0},
                {"matrix": [1, 0], "x": 1, "y": 0},
                {"matrix": [0, 1], "x": 2, "y": 0},
                {"matrix": [1, 1], "x": 3, "y": 0},
                {"matrix": [0, 2], "x": 4, "y": 0},
                {"matrix": [1, 2], "x": 5, "y": 0},
                {"matrix": [0, 3], "x": 6, "y": 0},
                {"matrix": [1, 3], "x": 7, "y": 0},
                {"matrix": [0, 4], "x": 8, "y": 0},
                {"matrix": [1, 4], "x": 9, "y": 0},
                {"matrix": [0, 5], "x": 10, "y": 0},
                {"matrix": [1, 5], "x": 11, "y": 0},
                {"matrix": [0, 6], "x": 12, "y": 0},
                {"matrix": [1, 6], "x": 13, "y": 0},
                {"matrix": [0, 7], "x": 14, "y": 0},
                {"matrix": [1, 7], "x": 15, "y": 0},
                {"matrix": [2, 0], "x": 0, "y": 1, "w": 1.5},
                {"matrix": [3, 0], "x": 1.5, "y": 1},
                {"matrix": [2, 1], "x": 2.5, "y": 1},
                {"matrix": [3, 1], "x": 3.5, "y": 1},
                {"matrix": [2, 2], "x": 4.5, "y": 1},
                {"matrix": [3, 2], "x": 5.5, "y": 1},
                {"matrix": [2, 3], "x": 6.5, "y": 1},
                {"matrix": [3, 3], "x": 7.5, "y": 1},
                {"matrix": [2, 4], "x": 8.5, "y": 1},
                {"matrix": [3, 4], "x": 9.5, "y": 1},
                {"matrix": [2, 5], "x": 10.5, "y": 1},
                {"matrix": [3, 5], "x": 11.5, "y": 1},
                {"matrix": [2, 6], "x": 12.5, "y": 1},
                {"matrix": [3, 7], "x": 15, "y": 1},
                {"matrix": [4, 0], "x": 0, "y": 2, "w": 1.75},
                {"matrix": [5, 0], "x": 1.75, "y": 2},
                {"matrix": [4, 1], "x": 2.75, "y": 2},
                {"matrix": [5, 1], "x": 3.75, "y": 2},
                {"matrix": [4, 2], "x": 4.75, "y": 2},
                {"matrix": [5, 2], "x": 5.75, "y": 2},
                {"matrix": [4, 3], "x": 6.75, "y": 2},
                {"matrix": [5, 3], "x": 7.75, "y": 2},
                {"matrix": [4, 4], "x": 8.75, "y": 2},
                {"matrix": [5, 4], "x": 9.75, "y": 2},
                {"matrix": [4, 5], "x": 10.75, "y": 2},
                {"matrix": [5, 5], "x": 11.75, "y": 2},
                {"matrix": [4, 6], "x": 12.75, "y": 2},
                {"matrix": [5, 6], "x": 13.75, "y": 1, "w": 1.25, "h": 2},
                {"matrix": [5, 7], "x": 15, "y": 2},
                {"matrix": [6, 0], "x": 0, "y": 3, "w": 1.25},
                {"matrix": [7, 0], "x": 1.25, "y": 3},
                {"matrix": [6, 1], "x": 2.25, "y": 3},
                {"matrix": [7, 1], "x": 3.25, "y": 3},
                {"matrix": [6, 2], "x": 4.25, "y": 3},
                {"matrix": [7, 2], "x": 5.25, "y": 3},
                {"matrix": [6, 3], "x": 6.25, "y": 3},
                {"matrix": [7, 3], "x": 7.25, "y": 3},
                {"matrix": [6, 4], "x": 8.25, "y": 3},
                {"matrix": [7, 4], "x": 9.25, "y": 3},
                {"matrix": [6, 5], "x": 10.25, "y": 3},
                {"matrix": [7, 5], "x": 11.25, "y": 3},
                {"matrix": [6, 6], "x": 12.25, "y": 3, "w": 1.75},
                {"matrix": [7, 6], "x": 14, "y": 3},
                {"matrix": [7, 7], "x": 15, "y": 3},
                {"matrix": [8, 0], "x": 0, "y": 4, "w": 1.5},
                {"matrix": [9, 0], "x": 1.5, "y": 4, "w": 1},
                {"matrix": [8, 1], "x": 2.5, "y": 4, "w": 1.5},
                {"matrix": [8, 3], "x": 4, "y": 4, "w": 7},
                {"matrix": [8, 5], "x": 11, "y": 4, "w": 1.5},
                {"matrix": [8, 6], "x": 13, "y": 4},
                {"matrix": [9, 6], "x": 14, "y": 4},
                {"matrix": [9, 7], "x": 15, "y": 4}
            ]
        }
    }
}
```

Reply:
Thank you! Really appreciate the thorough checking.